releasenotes/notes/process-comm-e0a76997f81d0542.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    During the process check, collect command name from `/proc/[pid]/comm`.
```

Comment:
Would be good to be more explicit about the intention here.

Reply:
What do you think about
> During the process check, collect command name from `/proc/[pid]/comm`. This allows more accurate language detection of processes.

Reply:
This is great, thank you 🙇